docs-src/ref-frontends-js.scrbl

```diff
@@ -115,6 +115,19 @@ In such scenarios, we recommend that you call this function like so:
 You may instead pass in the string @litchar{'ETH'} or the string @litchar{'ALGO'}
 to select the desired stdlib directly.
 
+By default, this method allows a user to load a single standard library.
+That is, this method may not be called multiple times, with the same or varying @tech{connectors}.
```

Comment:
Can you make it work on the same connector a second time?

Reply:
okay, sorry I misinterpreted the ticket